include/mbedtls/sha512.h

```diff
@@ -167,12 +173,12 @@ MBEDTLS_DEPRECATED static inline void mbedtls_sha512_update(
 }
 
 /**
- * \brief          SHA-512 final digest
+ * \brief          The SHA-512 final digest.
```

Comment:
Blocker: Please align the documentation of the deprecated functions to that of the superseding functions.

Reply:
ok